src/main/kotlin/no/ssb/metadata/vardef/integrations/vardok/services/StaticVardokService.kt

```diff
@@ -82,6 +83,7 @@ class StaticVardokService(
         result.otherLanguages.split(";").filter { it.isNotEmpty() }.forEach { l ->
             getVardokByIdAndLanguage(id, l).let { responseMap[l] = it }
         }
+        // Handle duplicate shortnames
```

Comment:
```suggestion
        // If we are attempting to migrate a variable with a short name which already exists in Vardef,
        // generate a new short name so that other metadata may be migrated.
```